.github/ISSUE_TEMPLATE/release_checklist.md

```diff
@@ -85,7 +85,7 @@ assignees: ''
 - [ ] update [the RPM repository](https://copr.fedorainfracloud.org/coprs/genericmappingtools/gmt/) (@seisman)
 - [ ] update the [try-gmt](https://github.com/GenericMappingTools/try-gmt) Jupyter lab (@seisman)
```

Comment:
@weiji14 Do you want to be added as the maintainer of the try-gmt repository?

```suggestion
- [ ] update the [try-gmt](https://github.com/GenericMappingTools/try-gmt) Jupyter lab (@weiji14, @seisman)
```

Reply:
Sure, go for it.